e2e-tests/e2e-tests.ts

```diff
@@ -28,17 +28,29 @@ function delay (timeout) {
 }
 
 describe('A non-matching journey', () => {
-  it('should succeed', async () => {
-    await journey().then(async function () {
-      const heading = await page.$eval('h2#heading-non-matching', e => e.innerHTML)
-      assert(heading.includes('Attributes for user with pid:'),'Actual: ' + heading)
+  describe('when eIDAS is not selected', () => {
```

Comment:
@alan-gds what do you think now? I added an extra `describe` so the code should read better and the output now looks like this: 
<img width="221" alt="image" src="https://user-images.githubusercontent.com/30629434/53171860-caeab480-35db-11e9-97ad-9dee3e8ae191.png">

Reply:
Looks great! 👍